modules/tinymce/src/core/main/ts/Rtc.ts

```diff
@@ -54,6 +55,7 @@ interface RtcRuntimeApi {
   editor: {
     getContent: (args: Partial<GetContentArgs>) => Content;
     setContent: (content: Content, args: Partial<SetContentArgs>) => Content;
+    addContentFormat: (format: string, formatGetter: GetContentFormatter, getCell: Cell<Record<string, GetContentFormatter>>, formatSetter: SetContentFormatter, setCell: Cell<Record<string, SetContentFormatter>>) => void;
```

Comment:
@TheSpyder @spocke are you okay with this API for the RTC integration?

Reply:
This is news to me, we haven't been made aware of this API or scheduled anything to manage it in 5.10.

Reply:
@TheSpyder this is for 6.0.0 at this stage (see the milestone), but I am surprised you haven't heard about it given this is a HV managed project...

Reply:
Ah maybe that's why. Can we get away without involving RTC in this content formatter? Looking at the `GetContentFormatter` API, custom formatters seem to be DOM specific?

Reply:
I don't know the details here, so tagging @lostkeys and @HAFRMO. If RTC don't need this though then I'm all for not doing this (I had assumed it was a requirement, probably incorrectly though).

Reply:
RTC has explicit implementations for tree, text, and HTML. Anything not tree or text produces HTML.

I don't see how we can make a content format API that works with RTC when the type is something like `(editor, args) => Content`. We'd need a public API that can abstract over the alt-model.

In theory `setContent` could run in front of RTC, since it's `content => content` but all a get content formatter could do is work from the ContentEditable DOM view (which isn't exactly what we render to a HTML string for get content).

If we want this to be RTC-compatible my suggestion is have both get and set formatters work from `AstNode` explicitly - RTC can deal with those.